CLEO4/CCustomOpcodeSystem.cpp

```diff
@@ -1928,8 +1934,8 @@ namespace CLEO
 	//0ACE=-1,show_formatted_text_box %1d%
 	OpcodeResult __stdcall opcode_0ACE(CRunningScript *thread)
 	{
-		char fmt[128	];
-		char text[128];
+		char fmt[128] = { 0 };
```

Comment:
Hmmm, why are those things needed?

Reply:
Not necessarily. I removed them in the new commit.